app/models/user.rb

```diff
@@ -2,4 +2,8 @@ class User < ApplicationRecord
   has_many :posts, foreign_key: :author_id
   has_many :likes
   has_many :comments
```

Comment:
- [ ] Since this is a has many relationship for Author with User as the class name, I will recommend you also add the author_id as foreign key for both likes and comments.